cpp/custrings/strings/NVStrings.cu

```diff
@@ -405,8 +405,6 @@ int NVStrings::create_offsets( char* strs, int* offsets, unsigned char* nullbitm
     unsigned int count = size();
     if( count==0 )
         return 0;
-    if( strs==0 || offsets==0 )
```

Comment:
If offsets is a null ptr don't we still want to early exit here?

Reply:
Oh, it will exit. It will exit in a hurry.

Reply:
I can look at adding an exception for this.